cmd/clusterctl/client/move.go

```diff
@@ -36,11 +38,21 @@ type MoveOptions struct {
 	// namespace will be used.
 	Namespace string
 
+	// Directory defines the local directory to store or restore cluster objects from
+	Directory string
+
+	// RestoreFromFolder restore configuration from folder. Used with Directory
+	RestoreFromFolder bool
+
+	// BackupToFolder save configuration to folder. Used with Directory
+	BackupToFolder bool
+
 	// DryRun means the move action is a dry run, no real action will be performed
 	DryRun bool
 }
 
-// BackupOptions holds options supported by backup.
+// BackupOptions holds options supported by backup. Deprecated and will be removed in a future release.
+// Please use MoveOptions.
```

Comment:
```suggestion
// BackupOptions holds options supported by backup. 
// Deprecated: This will be dropped in a future release. Please use MoveOptions.
```

nit. To be consistent with other deprecation messages that we have in code. 
the same for the deprecation notice we have on `RestoreOptions`.